zstd/plan.sh

```diff
@@ -1,14 +1,14 @@
 pkg_origin=core
 pkg_name=zstd
-pkg_version=1.3.8
+pkg_version=1.4.0
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('BSD-3-Clause')
 pkg_description="Zstandard is a real-time compression algorithm, providing high compression ratios. "\
 "It offers a very wide range of compression / speed trade-off, while being backed by a very fast decoder"
 pkg_upstream_url=http://facebook.github.io/zstd/
 pkg_source="https://github.com/facebook/zstd/archive/v${pkg_version}.tar.gz"
-pkg_shasum=90d902a1282cc4e197a8023b6d6e8d331c1fd1dfe60f7f8e4ee9da40da886dc3
-pkg_deps=(core/glibc)
+pkg_shasum='63be339137d2b683c6d19a9e34f4fb684790e864fee13c7dd40e197a64c705c1'
+pkg_deps=(core/glibc core/grep core/less)
```

Comment:
Lets split these onto single lines for easier reading and future diffs:

```
pkg_deps=(
  core/glibc
  core/grep
  core/less
)
```

Reply:
Good call, changed.